exec.go

```diff
@@ -0,0 +1,81 @@
+package gocql
+
+import (
+	"fmt"
+)
+
+// SingleHostQueryExecutor allows to quickly execute diagnostic queries while
+// connected to only a single node.
+type SingleHostQueryExecutor struct {
+	session *Session
+	control *controlConn
+}
+
+// Query executes the specified CQL statement.
+// Caller is responsible for closing the result iterator after use.
+func (e SingleHostQueryExecutor) Query(stmt string, values ...interface{}) *Iter {
```

Comment:
I would expect a method called `Query` to return `*Query`, similar to what `Session.Query()` does. Perhaps the method should be called `Execute` or `Exec` (`*Iter` has `Exec` method already)?

Reply:
Query has Exec and Iter, Iter does not have Exec.